dist/array_extensions.js

```diff
@@ -160,5 +160,20 @@
 
 		let cb = isFunction ? (a, b) => a + spec(b) : (a, b) => a + b;
 		return this.reduce(cb, 0);
+    }
+
+    function flatten(container=null) {
```

Comment:
The `container` parameter is not part of the specification on the API, we don't have to let doors open that can cause unwanted behaviors, unless you are planning to document that parameter you have to remove it and use a secondary or inner function to do the dirty work.